### PR TITLE
Tests for game functionalities

### PR DESCRIPTION
**Pull request by Angel-cuba**

### CODE REVIEW

File: cypress/e2e/games-management.cy.js [hunk 1]
```diff
@@ -74,12 +74,42 @@ describe('Games management', () => {
...
+      // Click update button and redirect to /
+      cy.findByRole('textbox', { name: 'Name' }).clear().type('example2')
+      cy.get('select').select('action')
+      cy.findByRole('textbox', { name: 'Year' }).clear().type(5678)
+      cy.findByRole('textbox', { name: 'Platform' }).clear().type('Ps3')
+      cy.findByRole('button', { name: 'Submit'}).click()
+      // Expect to be shown the new data
+      cy.rails('Routes.games_path').then(path => cy.visit(path))
 
+
```
Comment (jraunio) [inline cypress/e2e/games-management.cy.js:112]: Why are we not doing any assertions here?

File: cypress/e2e/games-management.cy.js [hunk 2]
```diff
@@ -74,12 +74,42 @@ describe('Games management', () => {
...
+      cy.findByRole('link', { name: 'Edit'}).click()
+
+      // Click update button and redirect to /
+      cy.findByRole('textbox', { name: 'Name' }).clear().type('example2')
+      cy.get('select').select('action')
+      cy.findByRole('textbox', { name: 'Year' }).clear().type(5678)
+      cy.findByRole('textbox', { name: 'Platform' }).clear().type('Ps3')
+      cy.findByRole('button', { name: 'Submit'}).click()
+      // Expect to be shown the new data
+      cy.rails('Routes.games_path').then(path => cy.visit(path))
```
Comment (jraunio) [inline cypress/e2e/games-management.cy.js:110]: We do we need this line?

Comment (Angel-cuba) [reply cypress/e2e/games-management.cy.js:110]: Actually we don't.

File: cypress/e2e/root-page.cy.js [hunk 3]
```diff
@@ -19,4 +19,71 @@ describe('Root page', () => {
...
+
+      // Check the different labels and link to delete the user
+      cy.findByRole('heading', { name: 'Cancel my account'}).should('exist')
+      cy.findByText('Unhappy?').should('exist')
+      // Click on the link to delete the user
+      cy.findByRole('button', { name: 'Cancel my account'}).click()
+
+      // Redirect to the root page
+      cy.findByText('Bye! Your account has been successfully cancelled. We hope to see you again soon.').should('be.visible')
+      cy.rails('Routes.games_path').then(path => cy.visit(path))
```
Comment (jraunio) [inline cypress/e2e/root-page.cy.js:83]: What is the purpose for this line?

File: cypress/e2e/games-management.cy.js [hunk 4]
```diff
@@ -17,4 +17,201 @@ describe('Games management', () => {
...
+      cy.rails(`Routes.new_game_path`).then(path => cy.visit(path))
+      // Check labels and inputs
+      cy.findByRole('textbox', { name: 'Name' }).type('example')
+      cy.get('select').select('action')
+      cy.findByRole('textbox', { name: 'Year' }).type(1234)
+      cy.findByRole('textbox', { name: 'Platform' }).type('Ps2')
+      cy.findByRole('button', { name: 'Submit'}).click()
+
+      // Click the DELETE button
+      // Expect to be redirected to /
```
Comment (jraunio) [inline cypress/e2e/games-management.cy.js:137]: These comments seem a bit random – can they be removed?

Comment (jraunio) [reply cypress/e2e/games-management.cy.js:137]: And there seem to be other random comments in other tests too. You could go through all the tests and decide which comments can be deleted.

**Review by jraunio — APPROVED** [review]

**Pull request merged**